faf-commons-lobby/src/test/kotlin/com/faforever/commons/lobby/ClientMessageTest.kt

```diff
@@ -0,0 +1,204 @@
+package com.faforever.commons.lobby
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule
+import com.fasterxml.jackson.module.kotlin.KotlinModule
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Test
+
+class ClientMessageTest {
+
+  companion object {
+    val objectMapper: ObjectMapper = ObjectMapper()
+      .registerModule(KotlinModule())
+      .registerModule(JavaTimeModule())
+  }
+
+  @Test
+  fun serializeHostGameRequest() {
+    assertEquals("{\"command\":\"game_host\",\"mapname\":\"name\",\"title\":\"title\",\"mod\":\"mod\",\"options\":[],\"access\":\"public\",\"version\":1,\"password\":null,\"visibility\":\"public\",\"rating_min\":null,\"rating_max\":null,\"enforce_rating_range\":false}",
```

Comment:
Use multiline string `""""` and proper json formatting. Also applies to the others below.

Reply:
Yep good point